source/client/main.qc

```diff
@@ -175,6 +175,9 @@ noref void(float apiver, string enginename, float enginever) CSQC_Init =
 	autocvar(in_rumbleenabled, 1);
 	autocvar(in_aimassist, 0);
 
+	cvar_set("show_speed_x", ftos(32));
+	cvar_set("show_speed_y", ftos(32));
```

Comment:
These positions will clash with other HUD elements, like Perk-A-Colas

Reply:
i changed it to `48, 48` for now, as i don't know the dimensions of that HUD element, but i'm happy to make the defaults anything.